### PR TITLE
Update filters to take a range

### PR DESCRIPTION
**Pull request by bytheway875**
Change SLA definition to six days, and allow the scope to take a range.

### CODE REVIEW

File: app/controllers/concerns/client_sortable.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def filtered_clients
     clients = clients.where(tax_returns: { service_type: @filters[:service_type] }) if @filters[:service_type].present?
     clients = clients.where(intake: Intake.where(had_unemployment_income: "yes")) if @filters[:unemployment_income].present?
     clients = clients.where(vita_partner: VitaPartner.allows_greeters) if @filters[:greetable].present?
-    clients = clients.first_unanswered_incoming_interaction_communication_breaches(@filters[:sla_breach_date]) if @filters[:sla_breach_date].present?
+    clients = clients.first_unanswered_incoming_interaction_between(...@filters[:sla_breach_date]) if @filters[:sla_breach_date].present?
```
Comment (alaroia-cfa) [inline app/controllers/concerns/client_sortable.rb:28]: `between` may be a little confusing. Maybe `first_unanswered_incoming_interaction_within` is a better name? Not a blocker.

File: app/models/client.rb [hunk 2]
```diff
@@ -96,8 +96,8 @@ def self.sortable_intake_attributes
   scope :with_eager_loaded_associations, -> { includes(:vita_partner, :intake, :tax_returns, tax_returns: [:assigned_user]) }
   scope :sla_tracked, -> { distinct.joins(:tax_returns).where.not(tax_returns: { status: TaxReturnStateMachine::EXCLUDED_FROM_SLA }) }
 
-  scope :first_unanswered_incoming_interaction_communication_breaches, ->(breach_threshold_datetime) do
-    sla_tracked.where(arel_table[:first_unanswered_incoming_interaction_at].lteq(breach_threshold_datetime))
+  scope :first_unanswered_incoming_interaction_between, ->(range) do
```
Comment (alaroia-cfa) [inline app/models/client.rb:99]: Given the relative simplicity of the scope, I think it's better to remove the scope entirely, and have the callsites directly query `Client.sla_tracked.where(first_unanswered_incoming_interaction_at: ...@breach_date)`. Bold suggestion, but that's what comes to mind. Not a blocker; I can live with the scope existing.

File: app/views/hub/metrics/index.html.erb [hunk 3]
```diff
@@ -8,7 +8,7 @@
       </div>
 
       <div class="spacing-above-15">
-        SLA Breach: Clients that haven't been contacted in 5 business days (since <%= long_formatted_datetime(@report.breached_at)%>)
+        SLA Breach: Clients that haven't been contacted in 6 business days (since <%= long_formatted_datetime(@report.breached_at)%>)
```
Comment (alaroia-cfa) [inline app/views/hub/metrics/index.html.erb:11]: Is this text change something product/design approves of? I guess it's an internal page anyway, but it's probably good to make sure they like whatever we end up with on this page. Not a blocker.

**Review by alaroia-cfa — APPROVED** [review]
Some questions, no blockers! Feel free to merge as-is or handle the comments.

File: app/models/client.rb (hunk 2 above)
Comment (bytheway875) [reply app/models/client.rb:99]: its called in multiple places, and it's important that the implementations are the same.

File: app/views/hub/metrics/index.html.erb (hunk 3 above)
Comment (bytheway875) [reply app/views/hub/metrics/index.html.erb:11]: The text change is accurate to the change they requested.

**Pull request merged**